src/macros.js

```diff
@@ -223,6 +223,17 @@ defineMacro("\\llap", "\\mathllap{\\textrm{#1}}");
 defineMacro("\\rlap", "\\mathrlap{\\textrm{#1}}");
 defineMacro("\\clap", "\\mathclap{\\textrm{#1}}");
 
+// Negated symbols from base/fontmath.ltx:
+// \def\neq{\not=} \let\ne=\neq
+// \DeclareRobustCommand
+//   \notin{\mathrel{\m@th\mathpalette\c@ncel\in}}
+// \def\c@ncel#1#2{\m@th\ooalign{$\hfil#1\mkern1mu/\hfil$\crcr$#1#2$}}
+defineMacro("\\neq", "\\not=");
+defineMacro("\\ne", "\\neq");
+defineMacro("\u2260", "\\neq");
+defineMacro("\\notin", "\\mathrel{{\\in}\\mathllap{/\\mskip1mu}}");
```

Comment:
Shouldn't the `/` be a `\not`?  It seems weird to use a different glyph for `\notin`.

Reply:
@kevinbarabash If I remember correctly, `\notin` was created shortly before `\not` was available.

Reply:
This is what LaTeX does; see the definition in comments. I believe it's a conscious decision because the `\in` glyph is relatively narrow. So I think we should follow suit.

Reply:
> They seem visually very close to LaTeX's; here's a texcmp for the tweaked Not test (only third line is new):

SGTM.

Reply:
...unless there's another LaTeX package that redefines `\notin` differently? I know AMS doesn't redefine it. I'm on a phone so can't easily check symbols-a4.

Reply:
There is a rather large variety of `\notin` symbols listed in [symbols-a4](http://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf), from packages `fdsymbol`, `stix`, `txfonts`, `mathabx`, `MnSymbol`.

![image](https://user-images.githubusercontent.com/2218736/40887216-297b53da-6713-11e8-8d52-1b8eca1505cf.png)

Not sure what to make of all this.  I guess I lean toward matching LaTeX's default behavior, but I'm also happy to design a `\not`-based macro if that's the preference.

Reply:
I'd lean towards `LaTeX`'s default behaviour as well.